sharedPrompts/docs/GITHUB_BODY_STORAGE_CHANGES.md

```
@@ -0,0 +1,33 @@
+| `src/main/java/.../module/domain/github/GitHubBodyStorageApplicationService.java` | 목록/미리보기/다운로드 애플리케이션 서비스 |
+| `src/main/java/.../module/controller/github/GitHubBodyStorageController.java` | GET /api/github/bodies, /{id}, /{id}/download API |
+| `src/main/java/.../module/dto/response/github/GitHubBodyStorageListDto.java` | 목록 응답 DTO |
+| `src/main/java/.../module/dto/response/github/GitHubBodyPreviewResponseDto.java` | 미리보기 응답 DTO (markdown) |
+| `src/main/java/.../module/dto/response/github/GitHubBodyDownloadUrlDto.java` | 다운로드 URL 응답 DTO |
+
+## 수정된 파일
+
+| 경로 | 변경 내용 |
+|------|----------|
+| `src/main/java/.../module/domain/github/GitHubBodyGenerateApplicationService.java` | `GithubBodyStorageRepository` 주입, `generate(promptId, request, tenantKey, eventType, ownerUserId)` 오버로드 추가, S3 저장 성공 후 `upsertMetadataIfNeeded()` 호출로 DB UPSERT |
+| `src/main/java/.../module/domain/github/GitHubWebhookHandlerService.java` | `handlePush` / `handlePullRequest`에서 `generate(..., tenantKey, "push"|"pull_request", config.getOwnerUserId())` 호출로 변경 |
```

markdownlint-cli2 (0.21.0)

[warning] Table column count
Expected: 2; Actual: 3; Too many cells, extra data will be missing

(MD056, table-column-count)